src/fcopy/main.c

```diff
@@ -286,7 +286,7 @@ static int fs_copydir(const char *infname, const struct stat *st, int ftype, str
 	}
 
 	// extract mode and ownership
-	if (stat(infname, &s) != 0)
+	if (lstat(infname, &s) != 0)
```

Comment:
Good catch!

Shouldn't this be done for `outfname` as well?

Reply:
This does not make any changes in behavior, but your right, it'll be more accurate.